.travis.yml

```diff
@@ -1,14 +1,22 @@
 language: python
 python:
   - "2.7"
-branches:
-  only:
-  - /.*/
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+    - os: osx
+      language: generic
 env:
   - SYSL_PLANTUML=http://www.plantuml.com/plantuml
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew list python &>/dev/null || brew install python; fi
```

Comment:
&> is a bash-ism and I cannot see anywhere here or in the travis docs that ensure these scripts are run with bash. It would be safer to do: >/dev/null 2>&1

Reply:
Done. I'm learning new stuff - thanks!